khmer/khmer_args.py

```diff
@@ -504,8 +504,14 @@ def calculate_graphsize(args, graphtype, multiplier=1.0):
         raise ValueError('unknown graph type: ' + graphtype)
 
     if args.max_memory_usage:
+        # subtract about 30MB for the python interpreter if user is requesting
+        # a large amount of memory that suggests they want to limit the
+        # total amount of RAM used for purposes of not running out of memory
+        max_memory = args.max_memory_usage
+        if max_memory > 1e8:
+            max_memory -= 3e7
```

Comment:
What do you think of this? If we really want to keep the total memory below the `-M` limit we need to subtract some memory for "the rest". This works if the max memory requested is large, but falls over if you set a limit of "10MB" or so. My guess is that in those cases the user is either testing or doing it for some reason that isn't limiting the true max memory used.